alot/commands/globals.py

```diff
@@ -78,7 +78,9 @@ def apply(self, ui):
 
         if ui.db_was_locked:
             msg = 'Database locked. Exit without saving?'
-            if (yield ui.choice(msg, msg_position='left', cancel='no')) == 'no':
+            if (yield ui.choice(msg,
+                                msg_position='left',
+                                cancel='no')) == 'no':
```

Comment:
I've been thinking about these, they're hard to read, but what if we took it out of the if statement?
```python
response = yield ui.choice(msg, msg_position='left', cancel='no')
if responce == 'no':
    ...
```

Reply:
works for me